source/Module/STIG/Functions.Checklist.ps1

```diff
@@ -592,34 +686,44 @@ function Get-FindingDetailsString
     }
     return $returnString
 }
+<#
+    .SYNOPSIS
+        Extracts the node targeted by the MOF file
+
+#>
 function Get-TargetNodeFromMof
 {
-    [OutputType([string])]
+    [OutputType([String])]
     [CmdletBinding()]
     param
     (
-        [Parameter(Mandatory)]
-        [string]
-        $MofString
+        [Parameter(Mandatory = $true)]
+        [String]
+        $mofString
     )
 
     $pattern = "((?<=@TargetNode=')(.*)(?='))"
-    $TargetNodeSearch = $mofstring | Select-String -Pattern $pattern
-    $TargetNode = $TargetNodeSearch.matches.value
-    return $TargetNode
+    $targetNodeSearch = $mofString | Select-String -Pattern $pattern
+    $targetNode = $targetNodeSearch.matches.value
+    return $targetNode
 }
+<#
+    .SYNOPSIS
+        Determines the type of node address
+
+#>
 function Get-TargetNodeType
 {
-    [OutputType([string])]
+    [OutputType([String])]
     [CmdletBinding()]
     param
     (
-        [Parameter(Mandatory)]
-        [string]
-        $TargetNode
+        [Parameter(Mandatory = $true)]
+        [String]
+        $targetNode
```

Comment:
Parameter capital letter (even if it's a private function)

Reply:
Done.